components/sup/src/util/toml.rs

```diff
@@ -0,0 +1,228 @@
+// Copyright (c) 2016-2017 Chef Software Inc. and/or applicable contributors
```

Comment:
This should just be `Copyright (c) 2016` - only ever need the date work first appeared.